ci-operator/step-registry/cucushift/installer/rehearse/azure/aks/hypershift/provision/cucushift-installer-rehearse-azure-aks-hypershift-provision-chain.yaml

```diff
@@ -0,0 +1,27 @@
+chain:
+  as: cucushift-installer-rehearse-azure-aks-hypershift-provision
+  steps:
+  - chain: cucushift-installer-rehearse-azure-aks-provision
+  - ref: cucushift-hypershift-extended-k8s-mgmt-apply-crds
+  - chain: cucushift-hypershift-extended-install-private
+  - ref: cucushift-hypershift-extended-k8s-mgmt-get-guest-annotations
+  - chain: hypershift-azure-create
+  - chain: cucushift-hypershift-extended-enable-qe-catalogsource
+  env:
+  - name: ENABLE_ICSP
+    default: "false"
+  - name: HYPERSHIFT_AZURE_LOCATION
```

Comment:
Maybe an old topic, why we do not use `LEASED_RESOURCE` here but introduce a customized region here? I can not remember the reason.

Reply:
Totally agree. I've already submitted a PR which default region to LEASED_RESOURCE for AWS by default. I also plan to do the same for Azure (in a separate PR). 

Thanks.